### ISSUE TITLE
No such property: splitsTask

### ISSUE THREAD

**Issue by andriichernenko** [bugsnag/bugsnag-android-gradle-plugin#83]
### Expected behavior
Android build completes normally.

### Observed behavior
Android build fails with the following error:
> No such property: splitsTask for class: com.bugsnag.android.gradle.BugsnagPlugin

### Steps to reproduce
1. Update bugsnag-android-gradle-plugin to version 3.2.3
2. Run `./gradlew buildEnvironment`

### Version
3.2.3

### Additional information
This problem does not occur on 3.2.2.
Here's the project's `build.gradle`:
```
buildscript {
    repositories {
        jcenter()
        maven {
            url 'https://maven.google.com/'
            name 'Google'
        }
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:3.0.1'
        classpath 'com.google.gms:google-services:3.0.0'
        classpath 'com.bugsnag:bugsnag-android-gradle-plugin:3.2.3'
    }
}

allprojects {
    repositories {
        mavenLocal()
        jcenter()
        maven {
            url "$rootDir/../node_modules/react-native/android"
        }
        maven {
            url 'https://maven.google.com/'
            name 'Google'
        }
    }
}

subprojects {
    afterEvaluate {project ->
        if (project.hasProperty("android")) {
            android {
                compileSdkVersion 27
                buildToolsVersion '27.0.2'
            }
        }
    }
}
```

**Comment by tonygriv** [comment]
+1, my build has blown up under my hands

**Comment by fractalwrench** [comment]
Thanks for the report - this should be resolved in v3.2.4.
